lib/src/components/NxThreatCounter/NxThreatCounter.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import classnames from 'classnames';
+
+import { Props, propTypes } from './types';
+export { Props } from './types';
+
+import './NxThreatCounter.scss';
+
+export default function NxThreatCounter({
+  criticalCount, severeCount, moderateCount, lowCount, layout, className, ...attrs }: Props) {
+
+  const layoutClasses = classnames('nx-threat-counter-container', className, {
+    [`nx-threat-counter-container--${layout}`]: layout
+  });
+
+  return (
+    <div className={layoutClasses} {...attrs}>
+      <dl className="nx-threat-counter nx-threat-counter--critical">
+        <dt className="nx-threat-counter__count">{criticalCount}</dt>
+        <dd className="nx-threat-counter__text">Critical</dd>
+      </dl>
+      <dl className="nx-threat-counter nx-threat-counter--severe">
+        <dt className="nx-threat-counter__count">{severeCount}</dt>
+        <dd className="nx-threat-counter__text">Severe</dd>
```

Comment:
You have the meaning of the `<dt>`s and `<dd>`s backwards.  "Severe" needs to be the `<dt>`.  However, the `<dt>` needs to appear in the DOM before the `<dd>` so you'll need to play with `flex-direction` too.

Reply:
> You have the meaning of the `<dt>`s and `<dd>`s backwards. "Severe" needs to be the `<dt>`. However, the `<dt>` needs to appear in the DOM before the `<dd>` so you'll need to play with `flex-direction` too.

https://github.com/sonatype/sonatype-react-shared-components/pull/343/commits/ed67c2380ec6f9e57c23a09aa7f893e0b765faf3